share/ci/backendFlags.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+###################################################
+# translate PIConGPU backend names into CMake Flags
+###################################################
+
+get_backend_flags()
+{
+    backend_cfg=(${1//:/ })
+    num_options="${#backend_cfg[@]}"
```

Comment:
why are you not using `backend_cfg[1]` as in the `result+=...` codes parts?

Reply:
Okay - this is just a moved old method - it was so already in the old code

Reply:
yes only moved the code to avoid copy-pasting it into all places where we use it.